### PR TITLE
FE playwright test: MH modal edits

### PR DESCRIPTION
**Pull request by Malar-Natarajan**
for: https://app.zenhub.com/workspaces/openstax-unified-5b71aabe3815ff014b102258/issues/openstax/unified/2282

MH edits: change color, add/edit note, delete highlight
updated config to run with 1 worker in local since running in parallel with multiple workers was failing

Note: The tests are still not having any asserts (will be in next PR)

### CODE REVIEW

File: playwright/src/fixtures/content.page.ts [hunk 1]
```diff
@@ -143,9 +146,21 @@ class ContentPage {
...
     // Scroll to top of content area and click
-    const body = await this.body.boundingBox()
-    await this.page.mouse.wheel(body.x + 100, body.y + 100)
-    await this.page.mouse.click(body.x + 100, body.y + 100)
+
+    const x1 = this.page.context().browser().browserType
+    console.log(x1)
+    
+    // console.log(test.info.name)
+
```
Comment (Malar-Natarajan) [inline playwright/src/fixtures/content.page.ts:154]: @gregfitch I want to get the browser name to do some browser specific actions here. Could you please let me know the right way to call for a browser name?

Comment (gregfitch) [reply playwright/src/fixtures/content.page.ts:154]: the easiest way is to use the `browserName` fixture ( https://playwright.dev/docs/api/class-fixtures#fixtures-browser-name )

Comment (Malar-Natarajan) [reply playwright/src/fixtures/content.page.ts:154]: @gregfitch do you mean like this in the test? https://github.com/openstax/rex-web/blob/dc5d473cea7bcae9c9449cc629f002d178d1a473/playwright/tests/rex-test/rex.behaviorspec.ts#L20

I am looking for something that can be called from the content.page.ts. Firefox is behaving slightly different after highlighting text, so I would want to do things diffently for firefox alone after highlighting text. Are there other options or can we somehow call that fixture inside the content.page.ts

Comment (gregfitch) [reply playwright/src/fixtures/content.page.ts:154]: In that case you may want to use either the javascript user agent property to get the user agent string (`window.navigator.userAgent` contains 'Gecko' or 'Firefox') or the browser type attached to the Browser Playwright object (https://playwright.dev/docs/api/class-browser#browser-browser-type is 'firefox').

File: playwright/src/fixtures/MHmodal.ts [hunk 2]
```diff
@@ -51,6 +79,55 @@ class MHHighlights {
...
+    // Click Cancel on note textbox
+   await this.cancelNote.click();
+  }
+
+  async clickSave() {
+    // Click Save on note textbox
+   await this.savelNote.click();
+  }
+
+  async clickDeleteHighlight(confirm: string) {
```
Comment (Malar-Natarajan) [inline playwright/src/fixtures/MHmodal.ts:121]: @gregfitch is there a way in Typescript to pass the values that can be accepted for a parameter?
In `clickDeleteHighlight(confirm: string)` I want the test to send values either `delete` or `confirm`. 
Can I add these 2 vaules in the function somehow?

**Comment by Malar-Natarajan** [comment]
@gregfitch this is ready for your review

File: playwright/src/fixtures/content.page.ts [hunk 3]
```diff
@@ -85,9 +90,8 @@ class ContentPage {
       await this.colorlocator.click()
     }
 
-    // click outside the highlighted paragraph to close the notecard
-    // Otherwise, the notecard can block other elements like next/previous links
-    await this.scrolltotop()
+    // Close notecard. Else, the it can block other elements like next/previous links.
```
Comment (gregfitch) [inline playwright/src/fixtures/content.page.ts:93]: you may want to clarify this comment

**Review by gregfitch — CHANGES_REQUESTED** [review]

**Comment by Malar-Natarajan** [comment]
@gregfitch updated the PR based on your review comments

File: playwright/src/fixtures/content.page.ts (hunk 3 above)
Comment (Malar-Natarajan) [reply playwright/src/fixtures/content.page.ts]: done

File: playwright/src/fixtures/MHmodal.ts [hunk 4]
```diff
@@ -51,6 +76,58 @@ class MHHighlights {
...
+      return this.blue.click()
+    } else if (color === 'green') {
+      return this.green.click()
+    } else if (color === 'pink') {
+      return this.pink.click()
+    } else if (color === 'purple') {
+      return this.purple.click()
+    } else if (color === 'yellow') {
+      return this.yellow.click()
+    }
```
[the remark this answers is not in the document]
Comment (Malar-Natarajan) [reply playwright/src/fixtures/MHmodal.ts]: done

**Review by gregfitch — APPROVED** [review]

**Pull request merged**